app/brave_settings_strings.grdp

```diff
@@ -609,6 +609,9 @@
 
     Are you sure you want to do this?
   </message>
+  <message name="IDS_BRAVE_SYNC_PASSPHRASE_DECRYPTION_ERROR_MESSAGE" desc="Warning message that informs user about Brave Sync is not running and browser needs to have Keychain access">
+    Browser could not decrypt the passphrase, Brave Sync is not running. Please check that browser has access to Keychain
```

Comment:
Maybe we can add something like "Don't use copied profile from other machines" because OSCrypt on Windows uses `CryptProtectData/CryptUnprotectData` (DPAPI) which tied to user login session. Using copied profile on MacOS would also fail either unless users manually modify `Brave Safe Storage` in keychain.

Reply:
Yes, this is also case.

I have updated the text https://github.com/brave/brave-core/compare/d7b1cbee5e6deee90c740668368230114459ab81..93a655a90634d318bab94643dbc3876a1075eb0a